exampleSite/content/index/items.md

```diff
@@ -0,0 +1,42 @@
++++
+fragment = "items"
+#disabled = false
+date = "2017-10-04"
+lastmod = "2017-10-04"
+weight = 100
+#background = "dark"
+selfLink = "items-fragment"
+
+title = "Items Fragment"
+subtitle= "Column based items with icons"
+
+[[items]]
```

Comment:
Each item might actually be nicer in it's own content file similar to how we do it in the member fragment. Then description can be done as .Content too. What do you think?

Reply:
Yeah, that'll look good. considering the fact we already  have a working PoC with the member fragment. I'll add that.